spec/index.html

```diff
@@ -1462,58 +1462,37 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
-          <li>Strings MUST be serialized with Unicode code points from <code class="codepoint">U+0000</code>
-            through <code class="codepoint">U+001F</code> using lower case hexadecimal Unicode notation
-            (`\uhhhh`) except for the set of predefined JSON control characters —
-            `U+0008`, `U+0009`, `U+000A`, `U+000C`, and `U+000D` — which SHOULD be
-            serialized as `\b`, `\t`, `\n`, `\f`, and `\r`, respectively. All
-            other Unicode characters SHOULD be serialized "as is", except
-            `U+005C` (`\`) and `U+0022` (`"`), which SHOULD be serialized as
-            `\\` and `\"`, respectively.</li>
-        </ul>
-        <div class="issue">The JSON Canonicalization Scheme (JCS) [[RFC8785]]
-          is an emerging standard for JSON canonicalization. This <code>rdf:JSON</code> specification
-          will likely be updated to require such a canonical representation.
-          Users are cautioned against depending on the lexical representation of
-          literals with the <code>rdf:JSON</code> datatype as RDF literals,
-          as the specifics of serialization may change in a future revision of
-          this document.</div>
-        Despite being defined as a set of strings, this value space is
-        considered distinct from the value space of <code>xsd:string</code>,
-        in order to avoid side effects with existing specifications.
-      </dd>
- 
+      <dd>is the set of
+        <a data-cite="RFC8259#section-7">strings</a>,
+        <a data-cite="RFC8259#section-6">numbers</a>,
+        <a data-cite="RFC8259#section-4">objects</a>,
+        <a data-cite="RFC8259#section-5">arrays</a>, and
+        literal values (`true`, `false`, and `null`) [[RFC8259]].
+
+        Two values are considered equal
+        if they are the same <a data-cite="RFC8259#section-7">string</a>,
+        <a data-cite="RFC8259#section-6">number</a>, or
+        literal value;
+        if they are both <a data-cite="RFC8259#section-5">arrays</a> with elements
+        which are pairwise equal;
+        or if they are both <a data-cite="RFC8259#section-4">objects</a> with equal members.
+      </dd> 
+
       <dt id="JSON-mapping">The <a>lexical-to-value mapping</a></dt>
-      <dd>
-        maps every element of the lexical space to the result of
-         <ol>
-          <li>parsing it into an internal representation consistent with the
-            [[ECMASCRIPT]] representation created by using the <code>JSON.parse</code>
-            function as defined in <a data-cite="ECMASCRIPT#sec-json-object">Section&nbsp;24.5 The JSON Object</a>
-            of [[ECMASCRIPT]],</li>
-          <li>then serializing it into the JSON format [[RFC8259]] in compliance
-            with the constraints of the value space described above.</li>
-        </ol>
+      <dd>maps every element of the lexical space to the result of parsing it into a
+        <a data-cite="RFC4627#section-2.1">JSON value</a>.
```

Comment:
So the value for "1.0"^^rdf:JSON is different from the value for "1.00"^^rdf:JSON.  This does not work.

Reply:
From my interpretation, they are different terms having the same value.

Reply:
Not so.   There is a missing jump from parsed JSON texts to some sort of data structure.   JSON strings, numbers, objects, and arrays are strings, or maybe parse trees.

Reply:
I'm not against the value space of rdf:JSON being recursively defined as the union of TRUE, FALSE, NULL, finite sequences of Unicode UTF 16 code units, the finite numeric part of IEEE double floating point numbers, finite sequences of value space elements, and finite partial maps from finite sequences of Unicode UTF 16 code units to value space elements but that is not what is being said here.